bidsify/utils.py

```diff
@@ -11,7 +11,7 @@
 import pydicom
 
 
-def run(command, env={}):
+def run(command, env=None):
```

Comment:
Can you make sure equivalent changes are merged into https://github.com/FIU-Neuro/cis-processing/pull/53?